### PR TITLE
Add result matching logic and pages

### PR DESCRIPTION
**Pull request by ttavenner**

### CODE REVIEW

File: api/services/SurveyResultService.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    get() {
+        return this.app.orm.SurveyResult.findAll();
+    }
+
+    match(params) {
+        return this.app.orm.SurveyResult.find({
+            where: params
+        })
+        .then((result) => {
+            const query = 'select surveyresult.id as "SurveyResultId"' +
```
Comment (qwo) [inline api/services/SurveyResultService.js]: we can leverage using ES6 strings to use multiline template 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: config/dev-fixtures/category-fixtures.js [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
-    })
-  }
-}
+    load: (app) => {
+        return app.orm.Category.count({})
+        .then(count => {
+            if (count > 0) {
+                return [];
+            }
+            else {
```
Comment (qwo) [inline config/dev-fixtures/category-fixtures.js]: unnecessary else since rest of statement is end

File: config/dev-fixtures/survey-fixtures.js [hunk 3]
```diff
@@ -1,38 +1,41 @@
...
-  }
+    load: (app) => {
+        return app.orm.Survey.count({})
+        .then(count => {
+            if (count > 0) {
+                return [];
+            }
+            else {
+                // Create surveys.
+                return Promise.all(
```
Comment (qwo) [inline config/dev-fixtures/survey-fixtures.js]: same unnecessary else

**Comment by ttavenner** [comment]
@stanzheng I made the three requested changes. I also finished wiring up the match service to the front end. It an be tested by hitting any of the fixture results (all currently identical):
http://localhost:3000/results/publicPhraseOne
http://localhost:3000/results/publicPhraseTwo
http://localhost:3000/results/publicPhraseThree

**Review by qwo — APPROVED** [review]

**Pull request merged**